sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -5537,6 +5568,61 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSparkSession {
     )
   }
 
+  test("json_array_length function") {
```

Comment:
Can these tests be placed in `JsonFunctionsSuite`?

Reply:
thanks, moved to `JsonFunctionsSuite`